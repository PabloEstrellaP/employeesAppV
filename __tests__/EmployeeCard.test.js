

import { mount } from '@vue/test-utils'
import EmployeeCard from '../components/EmployeesCard.vue' // ../my-component.vue file
import 'regenerator-runtime'

it('Render correctly', async() => {
  const wrapper = mount( EmployeeCard )
  // console.log(wrapper.findAll('pruebabotton'))
  const btnPrintE = wrapper.findAll('button')
  // await btnPrintE.trigger('click')
  const value = wrapper.find('#isTrue').text()
  expect( value ).toBe('false')
})
