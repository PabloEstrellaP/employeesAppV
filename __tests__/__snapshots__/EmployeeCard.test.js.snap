// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly 1`] = `
<div>
  <v-layout
    row=""
    wrap=""
  >
    <v-flex
      class="pa-5"
      md4=""
      xs12=""
    >
      <employeeaddcard-stub />
    </v-flex>
     
    <v-flex
      class="pa-5"
      md8=""
      xs12=""
    >
      <p
        id="isTrue"
      >
        false
      </p>
       
      <v-card>
        <v-toolbar
          flat=""
        >
          <v-toolbar-title>
            Employees
          </v-toolbar-title>
          <v-spacer />
        </v-toolbar>
         
        <v-card-text>
          <v-data-table
            headers="[object Object],[object Object],[object Object],[object Object],[object Object]"
            items=""
            items-per-page="5"
          />
        </v-card-text>
      </v-card>
    </v-flex>
  </v-layout>
   
  <v-dialog
    max-width="400"
    persistent=""
  >
    <v-card>
      <v-card-title
        class="display-1"
      >
        Loading...
      </v-card-title>
       
      <v-card-text
        class="display-5"
      >
        
      </v-card-text>
    </v-card>
     
    <v-progress-linear
      color="primary"
      indeterminate=""
    />
  </v-dialog>
   
  <employeedeletedialog-stub />
</div>
`;
